### ISSUE TITLE
cordova paypal plugin min-sdk problem

### ISSUE THREAD

**Issue by danielehrhardt** [phonegap/phonegap-plugin-barcodescanner#424]
Hello everyone 
thank you for that awesome plugin. I have a Problem with this plugin. 

### Expected Behaviour
Use this Plugin with other cordova plugins (Ionic Native).

### Actual Behaviour
Can't build my project (Android Build) with this plugin and the cordova paypal plugin
(https://github.com/paypal/PayPal-Cordova-Plugin)

> Manifest merger failed : uses-sdk:minSdkVersion 15 cannot be smaller than version 16 declared in library [com.paypal.sdk:paypal-android-sdk:2.15.3] /platforms/android/build/intermediates/exploded-aar/com.paypal.sdk/paypal-android-sdk/2.15.3/AndroidManifest.xml
  	Suggestion: use tools:overrideLibrary="com.paypal.android.sdk.payments" to force usage

### Reproduce Scenario (including but not limited to)
create a new ionic project and add both plugins 
```
cordova plugin add phonegap-plugin-barcodescanner --variable CAMERA_USAGE_DESCRIPTION="To scan barcodes"
cordova plugin add com.paypal.cordova.mobilesdk
```
and add `<preference name="android-minSdkVersion" value="16" />` to the config.xml

ionic build android

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
 ionic -v                                                            
2.2.1
npm -v
4.1.2
node -v   
v6.7.0
cordova --version                                                   
6.5.0
android 
6.1.0

#### Plugin version

    cordova plugin version | grep phonegap-plugin-barcodescanner
6.0.5

#### Logs taken while reproducing problem
/bitrise/src/platforms/android/AndroidManifest.xml:61:5-74 Error:
	uses-sdk:minSdkVersion 15 cannot be smaller than version 16 declared in library [com.paypal.sdk:paypal-android-sdk:2.15.3] /bitrise/src/platforms/android/build/intermediates/exploded-aar/com.paypal.sdk/paypal-android-sdk/2.15.3/AndroidManifest.xml
	Suggestion: use tools:overrideLibrary="com.paypal.android.sdk.payments" to force usage

**Comment by jcesarmobile** [comment]
Can you install the plugin from github url? you are installing 6.0.5 which doesn't have the latest changes to handle the minSDK properly

`cordova plugin add https://github.com/phonegap/phonegap-plugin-barcodescanner --variable CAMERA_USAGE_DESCRIPTION="To scan barcodes"`

**Comment by danielehrhardt** [comment]
BUILD SUCCESSFUL 
sorry for the Issue then please add a new Version
Thank you for your help :)

**Comment by macdonst** [comment]
@danielehrhardt I just released version 6.0.6.

**Comment by patrickbussmann** [comment]
@macdonst it's not working with 6.0.6 now.
I defined the min version in config.xml.

    <preference name="android-minSdkVersion" value="16" />

And using 6.0.6.

    <plugin name="phonegap-plugin-barcodescanner" spec="~6.0.6" />

And thats the error message.

	:processArmv7ReleaseManifest
	/bitrise/src/platforms/android/AndroidManifest.xml:61:5-74 Error:
		uses-sdk:minSdkVersion 15 cannot be smaller than version 16 declared in library [com.paypal.sdk:paypal-android-sdk:2.15.3] /bitrise/src/platforms/android/build/intermediates/exploded-aar/com.paypal.sdk/paypal-android-sdk/2.15.3/AndroidManifest.xml
		Suggestion: use tools:overrideLibrary="com.paypal.android.sdk.payments" to force usage

Don't know why this is so complicated :laughing:
